app/javascript/components/UserDisplay/Dashboard/Collection/index.jsx

```diff
@@ -37,7 +37,7 @@ const Collection = ({
           {filteredCollection.map((card, row) => (
             <Row>
               <Cell isPriority>
-                <a href={`/cards/${card.id}`}>{card.name}</a>
+                <a className="invert" href={`/cards/${card.id}`}>{card.name}</a>
```

Comment:
Lets change the link color to the invert color here, and update the nav links to have a different color